### DISCUSSION TITLE
Multiple search results for the same item/site

### DISCUSSION THREAD

**Discussion by Robby7979** [microsoft-search/pnp-modern-search#2438] [Q&A] [answered]
We have a pnp-modern-search that searches our "client list" in SPO. In the last couple of weeks we have been getting reports of multiples of those sites are showing in the search results. Each result goes to the same link, but this provides our users with confusion on which one to use. We are using pnp-modern-search 4.4.10.  We had a meeting with Microsoft today, and they said the SharePoint search is working fine and its likely a issue with the webpart. Has anyone else had this experience or have suggestions to try?

**Comment by wobba** [comment, marked as answer]
Well, if it's been working all along, I doubt it's a web part issue. I think you need to explain a bit more and add more details aroun your scenario "client list" and "sites" to understand if you have a KQL issue, or if it's a permission issue on items now surfacing etc.

**Comment by Robby7979** [comment]
Sorry I did not provide enough info, I was unable to upload a picture as well as I am new to my role with the company. The search part just searches a list of "clients" we have with our company. Using the standard SharePoint search only a single result appears but when using the modern search duplicates show up, however this is not occuring in every client.

![IMG_20221017_103602593_HDR](https://user-images.githubusercontent.com/115585200/196220847-fd6b47a7-c602-4499-815e-409c255bd47f.jpg)

**Reply by wobba** [reply]
When you say "standard SharePoint search", what do you mean? And the OOB search UX in SharePoint at the site level will not trim duplicates (as the feature is not a good one the way it's implemented - too many false positives)

You can turn duplicate trimming on/off in the web part.

![image](https://user-images.githubusercontent.com/51104/197389718-06d3a101-d24a-403a-be6a-98df25c5182a.png)

Without knowing what your query is against or what you expect to get back it's hard to say what happens.

**Comment by Robby7979** [comment]
Thanks Miakel,

Yes the standard SharePoint search does not re-produce the duplicate results, but when we use Modern Search it does show duplicate results. At this point we are still on a older version of Modern Search that does not have the trim duplicates option, we are In process on getting the update tested and pushed out for production. Hopefully that Trim duplicate feature will take care of this. I will follow up if it does not. Thanks!


Rob Selvidge
SharePoint Administrator I  /  Firm Technology Services
D: 417.408.8700
www.linkedin.com/in/robselvidge<http://www.linkedin.com/in/robselvidge>
***@***.***<forvis.com>  ***@***.*** <linkedin.com/company/forvis/about/>  ***@***.*** <twitter.com/FORVIS>  ***@***.*** <facebook.com/FORVISLLP/>  ***@***.*** <instagram.com/forvis_llp/>  ***@***.*** <youtube.com/channel/UCiSZDw509f_2TSSWS_Yl9xw>
